#!/usr/bin/python3
def raise_exception_msg(message=""):
    """function that raises a name error
    Args:
        message: message to be printed after raising the error
    Returns:
        None
    """
    raise NameError(message)
